high-scores/high_scores.rb

```diff
@@ -0,0 +1,37 @@
+=begin
+Write your code for the 'High Scores' exercise in this file. Make the tests in
+`high_scores_test.rb` pass.
+
+To get started with TDD, see the `README.md` file in your
+`ruby/high-scores` directory.
+=end
+class HighScores
+  #intente class HighScores < Struct.new(:scores)
+  #tambien attr_accessor(:scores)
+    def initialize(scores)
```

Comment:
Be consistent with the indentation. You're using 2 spaces for indentation of the comments and 4 for the rest of the code. Pick one and always use that one.